### PR TITLE
Adds documentation and analytics example for Magic Transit tunnel hea…

### PR DESCRIPTION
**Pull request by govind-menon**
…lthchecks

### CODE REVIEW

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: Querying Magic Transit Tunnel Health C3heck Results with GraphQL
```
Comment (annikagarbers) [inline products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md]: tiny typo in "Check" here & on line 6

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+order: 50
+---
+
+# Querying Magic Transit Tunnel Health C3heck Results with GraphQL
+
+In this example, we're going to use the GraphQL Analytics API to query Magic Transit Healthcheck Results which are aggregated from individual health checks carried out by Cloudflare servers to GRE tunnels you've setup to work with Magic Transit during the [onboarding process](https://developers.cloudflare.com/magic-transit/set-up/onboarding). We can query up to one week of data for dates up to three months ago.
```
Comment (annikagarbers) [inline products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md]: "Healthcheck Results" -> "health check results"; "setup" -> "set up"

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+The following API call will request a particular account's tunnel health checks over a one day period for a particular Cloudflare colo, and outputs the requested fields. Be sure to replace `CLOUDFLARE_EMAIL` and `CLOUDFLARE_API_KEY` with your email and API credentials, and adjust the `datetimeStart`, `datetimeEnd` and `accountTag` variables as needed.
+
+## API Call
+
+```
+CLOUDFLARE_EMAIL=<CLOUDFLARE_EMAIL>
+CLOUDFLARE_API_KEY=<CLOUDFLARE_API_KEY>
+PAYLOAD='{ "query":
+  "query GetWorkersAnalytics($accountTag: string, $datetimeStart: string, $datetimeEnd: string, $scriptName: string) {
```
Comment (annikagarbers) [inline products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md]: guessing you will update this tomorrow but should this be something other than GetWorkersAnalytics here?

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+      viewer {
+        accounts(filter: {accountTag: $accountTag}) {
+          conduitEdgeTunnelHealthChecks(
```
Comment (annikagarbers) [inline products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md]: is the "conduit edge" terminology in the customer facing api?

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+      "viewer": {
+        "accounts": [
+          {
+            "conduitEdgeTunnelHealthChecks": [
+              {
+                "count": 180,
+                "dimensions": {
+                  "edgeColoName": "MEL",
+                  "tunnelName": "tunnel_01",
+                  "tunnelState": 0.5
```
Comment (annikagarbers) [inline products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md]: can we put a note somewhere about what the tunnelState numbers mean?

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md (hunk 4 above)
Comment (govind-menon) [reply products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md]: It is - it's in the graphql endpoint

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+PAYLOAD='{ "query":
+  "query GetTunnelHealthCheckResults($accountTag: string, $datetimeStart: string, $datetimeEnd: string, $scriptName: string) {
+    {
+      viewer {
+        accounts(filter: {accountTag: $accountTag}) {
+          magicTransitTunnelHealthChecksAdaptiveGroups(
+            limit: 100,
+            filter: {
+              datetime_geq: $datetimeStart,
+              datetime_lt:  $datetimeEnd,
```
Comment (soheiokamoto) [inline products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md:28]: Don't know if it makes in invalid or not, but looks like you have extra `,` for the last one.

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+            }
+          }
+        }
+      }
+    }
+  }",
+    "variables": {
+      "accountTag": "90f518ca7113dc0a91513972ba243ba5",
+      "datetimeStart": "2020-05-04T00:00:00.000Z",
+      "datetimeEnd": "2020-05-04T00:00:00.000Z",
```
Comment (soheiokamoto) [inline products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md:46]: Don't know if it makes in invalid or not, but looks like you have extra `,` for the last one.

**Review by tlozoot — APPROVED** [review]

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md (hunk 7 above)
Comment (govind-menon) [reply products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md:46]: That should be fine

File: products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+The results returned will be in JSON (as requested), so piping the output to `jq` will make them easier to read, e.g.,:
+
+```
+curl \
+  -X POST \
+  -H "Content-Type: application/json" \
+  -H "X-Auth-Email: CLOUDFLARE_EMAIL" \
+  -H "X-Auth-key: CLOUDFLARE_API_KEY" \
+  --data "$(echo $PAYLOAD)" \
```
Comment (jherre) [inline products/analytics/src/content/graphql-api/tutorials/querying-magic-transit-tunnel-healthcheck-results/index.md:67]: This `"$(echo $PAYLOAD)"` is a strange construct but it works...

**Review by jherre — APPROVED** [review]

**Pull request merged**